### PR TITLE
Introduce new bridge impl

### PR DESCRIPTION
**Pull request by spring2maz**
NOTE: This is still a work-in-progress PR.

### CODE REVIEW

File: priv/emqx.schema [hunk 1]
```diff
@@ -1607,16 +1596,23 @@ end}.
   {datatype, {duration, ms}}
 ]}.
 
-{mapping, "bridge.$name.retry_interval", "emqx.bridges", [
-  {default, "20s"},
-  {datatype, {duration, ms}}
-]}.
-
```
Comment (Gilbert-Wong) [inline priv/emqx.schema]: Why did you delete this config entry ?

Comment (spring2maz) [reply priv/emqx.schema]: there is no more delay-retry for sending messages, rather a delay-retry of reconnect, messages to retry will be sent right after a successful reconnect.

Comment (spring2maz) [reply priv/emqx.schema]: see `bridge.$name.reconnect_interval`

File: include/emqx_client.hrl [hunk 2]
```diff
@@ -1,4 +1,4 @@
-%% Copyright (c) 2013-2019 EMQ Technologies Co., Ltd. All Rights Reserved.
+%% Copyright (c) 2019 EMQ Technologies Co., Ltd. All Rights Reserved.
```
Comment (turtleDeng) [inline include/emqx_client.hrl]: %% Copyright (c) 2013-2019 EMQ Technologies Co., Ltd. All Rights Reserved.

File: src/emqx_portal_connect.erl [hunk 3]
```diff
@@ -0,0 +1,71 @@
+%% Copyright (c) 2019 EMQ Technologies Co., Ltd. All Rights Reserved.
```
Comment (turtleDeng) [inline src/emqx_portal_connect.erl]: %% Copyright (c) 2013-2019 EMQ Technologies Co., Ltd. All Rights Reserved.
